code/game/machinery/constructable_frame.dm

```diff
@@ -754,6 +754,15 @@ obj/item/weapon/circuitboard/rdserver
 							"/obj/item/weapon/stock_parts/matter_bin" = 1,
 							"/obj/item/weapon/stock_parts/capacitor" = 2)
 
+/obj/item/weapon/circuitboard/box_cloner
+	name = "Circuit Board (Box Cloner)"
+	build_path = "/obj/machinery/egg_incubator/box_cloner"
+	origin_tech = "illegal=3"
```

Comment:
If the mechanics want to trade for the circuitboard, duplicate it, and gain illegal tech that way, that seems like good gameplay design, not breaking the flow.

As if people wouldn't just do the normal way of getting illegal tech from the wiki - the tesla relays